src/configuration/GPG/GnuPG/gpg.conf

```diff
@@ -205,8 +205,7 @@ keyserver hkp://keys.gnupg.net
 # photo-viewer "metamail -q -d -b -c %T -s 'KeyID 0x%k' -f GnuPG"
 
 
-personal-digest-preferences SHA256
-cert-digest-algo SHA256
-default-preference-list SHA512 SHA384 SHA256 SHA224 AES256 AES192 AES CAST5 ZLIB BZIP2 ZIP Uncompressed
-
+personal-digest-preferences SHA512
+cert-digest-algo SHA512
+default-preference-list AES256, CAMELLIA256, AES192, CAMELLIA192, AES, CAMELLIA128, TWOFISH, SHA512, SHA384, SHA256, BZIP2, ZLIB, ZIP 
```

Comment:
>  I think you are right that these algorithms are rarely used but I'm not aware that one of them is not secure anymore.

The name of this project is *bettercrypt* not *notunsecurecrypto* :D 

And I also think that AES128 and similar are still better then 3DES (see also the manpage of gpg)